src/interfaces/python/swig_typemaps.i

```diff
@@ -848,89 +848,6 @@ static bool spvector_to_numpy(PyObject* &obj, SGSparseVector<type> sg_vector, in
 	}
 %enddef
 
-%typemap(out) shogun::CFeatures*
```

Comment:
since the features are only returned by base class in the future, we don't really need this anymore

I guess there is a question here: Are feature types part of the core shogun or will they be plugins @lisitsyn @vigsterkr ?
The current approach in the new examples handles all features by `CFeatures` only, but I now just started to wonder whether we really want to do that? Certainly reduces the number of types ... (`RealFeatures, FloatFeatures, ...` for dense, string, sparse each)

Reply:
going with making them plugins for now ...